src/utils.ts

```diff
@@ -173,10 +173,28 @@ export async function getScores(
   }
 }
 
+function isUrl(str) {
+  if (!str.length) return true;
```

Comment:
How is an empty string a URL? Shouldn't it return false then?

Reply:
EIP712 needs it to be a string otherwise it will throw an error. Yeah I will try to inline it.